.github/workflows/test.yml

```diff
@@ -7,6 +7,10 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
```

Comment:
It seems like we already have this on `main` branch https://github.com/typeddjango/django-stubs/blob/4a000e83ff0ce82e66c97178fc034ea939113a8a/.github/workflows/test.yml#L10

Reply:
See https://github.com/typeddjango/django-stubs/pull/2703#issuecomment-2943485106